com.microsoft.mrtk.uxcore/SeeItSayIt/SeeItSayItLabelEnabler.cs

```diff
@@ -47,6 +50,12 @@ private void Start()
             {
                 // Check if input and speech packages are present
 #if MRTK_INPUT_PRESENT && MRTK_SPEECH_PRESENT
+                // If we can't find any active speech interactors, then do not enable the labels.
+                if (!Object.FindAnyObjectByType<SpeechInteractor>())
```

Comment:
This feels like it could get expensive on start-up with large numbers of see-it, say-it labels, if all of them are searching the entire scene hierarchy.

This feels like a design-time failure case, if I have a see-it, say-it label configured but no speech interactor present, as opposed to something we should protect against at runtime.

Reply:
Good point... @marlenaklein-msft may be we can create our own version of `ComponentLocatorUtility`.   Unity XRI uses to `ComponentLocatorUtility<T>` to cache common object query results (like XRInteractionManager).  Something like this might be useful.

Reply:
Sounds good, I will remove this check.

Reply:
Chatted offline,  we'll remove this change for now and revisit with a new PR